Create a migration guide (#6210)
kyma-project/kyma: docs/migration-guides/1.5-1.6.md

# Migrate from v1.5 to v1.6

>**TIP:** To learn more about upgrading Kyma, read [this](https://kyma-project.io/docs/master/root/kyma/#installation-upgrade-kyma) document.

## Service Management

> We have migrated the Service Catalog to the CRDs in this [PR](https://github.com/kubernetes-sigs/service-catalog/pull/2630). To support upgrade from earlier versions, we created a migration job which migrates their Service Catalog related resources on upgrade.

Reviewer: ```suggestion
We have migrated the Service Catalog API Server to the new CRDs approach. See [this](https://github.com/kubernetes-sigs/service-catalog/pull/2630) PR for more details. To support upgrade from the earlier Kyma versions, we created a migration job which migrates your Service Catalog-related resources.
```

> In order to make a successful upgrade of Kyma, the Service Catalog resources can't be in the unfinished/deleted state. Otherwise our upgrade job can fail.

Reviewer: ```suggestion
In order to perform the successful Kyma upgrade, the Service Catalog resources can't be in the unfinished or deleted state. Otherwise, the upgrade job can fail.
```

Reviewer: I miss the information here. What the user has to do exactly?

> To check if your cluster is ready to migration, use the sanity check [script](https://github.com/kubernetes-sigs/service-catalog).

Reviewer: ```suggestion
To check if your cluster is ready for migration, use the [sanity check script](https://github.com/kubernetes-sigs/service-catalog).
```

> >**CAUTION:** You can safely remove the migration job PVC, which contains data about your Service Catalog resources only when the migration end up successfully.

Reviewer: ```suggestion
>**CAUTION:** You can safely remove the migration job PVC that contains data about your Service Catalog resources only when the migration ends up successfully.
```
